src/node.cpp

```diff
@@ -1073,8 +1073,11 @@ void LocalNode::setnameparent(LocalNode* newparent, string* newlocalpath)
         }
     }
 
-    LocalTreeProcUpdateTransfers tput;
-    sync->client->proclocaltree(this, &tput);
+    if (newlocalpath)
```

Comment:
If the path of the LocalNode hasn't changed, there is no need to update related Transfers. Also, that can cause crashes in some corner cases.